XenAdmin/MainWindow.cs

```diff
@@ -1704,6 +1704,10 @@ private void pluginManager_PluginsChanged()
                 }
 
                 // get insert index using the placeholder
+
+                if (!pluginMenuItemStartIndexes.ContainsKey(menu))

```

Comment:
This is needed because the addition of the new item on the top Menu caused a crash when opening the Options dialog and hitting OK.